src/app/[locale]/_utils/common.ts

```diff
@@ -40,3 +42,12 @@ export const getLoginFlow = (loginInfo: LoginInfo): string => {
     return '';
   }
 };
+
+export const getReferralLockProfile = (isMagicLink: MagicLink): string => {
+  storageMagicLinkOps.delete();
```

Comment:
As we discussed earlier in another PR, putting a delete in a `get` method could lead a misunderstanding and critical paths in the long term. What do you think?